docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Controller Deployment

```

Comment:
After looking at this for a bit, it felt redundant to see "Deployments -> Linux -> Controller Deployment" and the sidebar also shows just "Controller". I think "Deployment" here is redundant and can/should be removed.

I'd also like to see the preamble here to indicate the three phases (are there more?) of deployment:
* Installation/Package installation
* Configuration
* Starting/Running as a service

Reply:
The title should be descriptive without being too long. The title is separate from the breadcrumb/sidebar hierarchy, i.e., the title stands alone. Conversely, the `id` (slug) is part of the breadcrumb (in the context of sidebar labels) and shouldn't be redundant, so the `id: deploy` should have been just `index`.